starport-core/src/main/scala/com/krux/starport/metric/MetricSettings.scala

```diff
@@ -2,126 +2,89 @@ package com.krux.starport.metric
 
 import java.util.concurrent.TimeUnit
 
-import scala.util.Random
 import scala.collection.JavaConverters._
-import com.codahale.metrics.graphite.{Graphite, GraphiteReporter}
-import com.codahale.metrics.{ConsoleReporter, MetricFilter, MetricRegistry, ScheduledReporter}
-import io.github.azagniotov.metrics.reporter.cloudwatch.CloudWatchReporter
-import io.github.azagniotov.metrics.reporter.cloudwatch.CloudWatchReporter.Percentile
+import scala.util.Random
+
 import com.amazonaws.regions.Regions
 import com.amazonaws.services.cloudwatch.model.StandardUnit
 import com.amazonaws.services.cloudwatch.{AmazonCloudWatchAsync, AmazonCloudWatchAsyncClientBuilder}
+import com.codahale.metrics.graphite.{Graphite, GraphiteReporter}
+import com.codahale.metrics.{ConsoleReporter, MetricFilter, MetricRegistry, ScheduledReporter}
 import com.typesafe.config.Config
+import io.github.azagniotov.metrics.reporter.cloudwatch.CloudWatchReporter
+import io.github.azagniotov.metrics.reporter.cloudwatch.CloudWatchReporter.Percentile
 
-/**
- * A service trait that is used by private case class constructors.
- */
-trait MetricSettingsImpl extends Serializable {
-
-  def getReporter(registry: MetricRegistry): ScheduledReporter
-
-}
 
-/**
```

Comment:
Any reason why this comment should also be removed? Is it because `sealed trait` is self-explanatory?

Reply:
Yeah, seems unnecessary here

Reply:
Got it.